client/package.json

```diff
@@ -12,7 +12,12 @@
     "number-to-words": "^1.2.4",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.4.0"
+    "react-redux": "^7.2.0",
+    "react-router-dom": "^5.1.2",
+    "react-scripts": "3.4.0",
+    "redux": "^4.0.5",
+    "redux-devtools-extension": "^2.13.8",
```

Comment:
shouldn't this be in dev-dependencies?

Reply:
I left that intentionally. For now it will help us in finding bugs which affect only production, for some reason. To not allow on production, we can change the import to 
```
import { composeWithDevTools } from "redux-devtools-extension/developmentOnly"
```